tests/deploy_test.go

```diff
@@ -194,6 +345,25 @@ func Test_Deploy_WithAnnotations(t *testing.T) {
 	}
 }
 
+func scale(t *testing.T, function string, replicas uint64) (int, error) {
+
+	req := map[string]interface{}{"services": function, "replicas": replicas}
+
+	body, res, err := httpReq(fmt.Sprintf("%ssystem/scale-function/%s",
+		os.Getenv("gateway_url"), function), http.MethodPost, makeReader(req))
+
+	if err != nil {
+		return http.StatusBadGateway, err
+	}
+
+	if res.StatusCode >= 400 {
```

Comment:
We should get a specific response from this only. Accepted ideally, possibly OK.